docs/githubteacher.md

```diff
@@ -0,0 +1,2 @@
+# GitHub Teacher Bio
+- Name: GitHub Teacher
```

Comment:
There should be more information to add here that is found in the corresponding issue.

More coming soon!